static/src/javascripts-legacy/projects/common/modules/experiments/tests/bundle-digital-sub-price-test-1.js

```diff
@@ -4,128 +4,81 @@ define([
-        this.audienceCriteria = 'Non-paying UK network front users - mobile resolution and above';
-        this.dataLinkNames = '';
-        this.idealOutcome = 'Find the price that works for most people.';
-        this.hypothesis = 'One of our price points will be more desirable than the others';
-
-        this.canRun = function () {
-            return document.querySelector('#membership-ab-thrasher') &&
-                    !cookies.getCookie('GU_DBPT1M') &&
-                    !userFeatures.isPayingMember() &&
-                    config.page.isFront &&
-                    config.page.pageId.toLowerCase() === "uk" &&
-                    config.page.edition.toUpperCase() === 'UK';
-        };
-
-        this.thrasherContainer = function() {
-            return document.querySelector('#membership-ab-thrasher');
-        };
-
-        this.thrasher = function() {
-            return document.querySelector('.membership-ab-thrasher--wrapper');
-        };
-
-        this.setTitle = function() {
-            if (this.thrasher()) {
-                var titleEl = document.querySelector('.membership-ab-thrasher_header .main_title');
-                if (titleEl) {
-                    titleEl.innerHTML = '<p>Support' + (detect.isBreakpoint({min: 'mobileMedium', max: 'tablet'}) ? ' ' : '<br>')  + 'the Guardian</p>';
-                }
-            }
-        };
-
-        this.setCopy = function() {
-            if (this.thrasher()) {
-                var subTitleEl = document.querySelector('.membership-ab-thrasher_header .sub_title');
-                if (subTitleEl) {
-                    subTitleEl.innerHTML = '<p>We\'re introducing <strong>new ways</strong> to support' + (detect.isBreakpoint({min: 'desktop'}) ? '<br>' : ' ') + 'the Guardian\'s quality journalism and independent voice. Choose to subscribe or contribute today.</p>';
-                }
-            }
-        };
-
-        this.setLink = function(variant) {
-            if (this.thrasher()) {
-                var linkEl = document.querySelector('.membership-ab-thrasher--wrapper .link-button');
-                if (linkEl && linkEl.getAttribute('href')) {
-                    linkEl.setAttribute('href', config.page.membershipUrl + '/bundles?INTCMP=BUNDLE_PRICE_TEST_1M_' + config.page.edition.toUpperCase() + '_' + variant.toUpperCase());
-                }
-            }
-        };
-
-        this.setCaption = function() {
-            if (this.thrasher()) {
-                var captionEl = document.querySelector('.membership-ab-thrasher--wrapper .link-button div p');
-                if (captionEl) {
-                    captionEl.textContent = 'Subscribe now';
-                }
-            }
-        };
+    var buildTemplate = function(variant) {
+        return template(contributionsEpicSingleButton, {
+            linkUrl1: config.page.membershipUrl + '/bundles?INTCMP=' + 'BUNDLE_PRICE_TEST_1M_' + config.page.edition.toUpperCase() + '_' + variant.id.toUpperCase(),
```

Comment:
Infact, as you are saying that the test is only for the UK edition, you don't need to encode the edition into the campaign code. You can replace this with variant.getURL(“https://www.theguardian.com/membership/bundles“, variant.options.campaignCode)

and set the campaign ID for the test to "BUNDLE_PRICE_TEST_1M"

This will mean your campign code will come out as gdnwb_copts_memco_BUNDLE_PRICE_TEST_1M_A, gdnwb_copts_memco_BUNDLE_PRICE_TEST_1M_B, 
gdnwb_copts_memco_BUNDLE_PRICE_TEST_1M_C